lms/djangoapps/teams/static/teams/js/views/topic_teams.js

```diff
@@ -24,51 +52,48 @@
+                            teamset_id: this.model.get('id')
+                        }
+                    }).done(function(data) {
+                        deferred.resolve(data.count === 0);
+                    }).fail(function(data) {
+                        TeamUtils.parseAndShowMessage(data, canUserCreateTeamErrorMessage);
+                        deferred.resolve(false);
+                    });
+                }
+                return deferred.promise();
+            },
+
+
+            showActions: function() {
+                HtmlUtils.append(
+                    this.$el,
+                    HtmlUtils.template(teamActionsTemplate)({message: actionMessage})
+                );
             },
 
             render: function() {
-                var self = this;
-                this.collection.refresh().done(function() {
-                    var message;
-                    TeamsView.prototype.render.call(self);
-                    if (self.canUserCreateTeam()) {
-                        message = interpolate_text(  // eslint-disable-line no-undef
-                                // Translators: this string is shown at the bottom of the teams page
-                                // to find a team to join or else to create a new one. There are three
-                                // links that need to be included in the message:
-                                // 1. Browse teams in other topics
-                                // 2. search teams
-                                // 3. create a new team
-                                // Be careful to start each link with the appropriate start indicator
-                                // (e.g. {browse_span_start} for #1) and finish it with {span_end}.
-                                _.escape(gettext(
-                                    '{browse_span_start}Browse teams in other ' +
-                                    'topics{span_end} or {search_span_start}search teams{span_end} ' +
-                                    'in this topic. If you still can\'t find a team to join, ' +
-                                    '{create_span_start}create a new team in this topic{span_end}.'
-                                )),
-                            {
-                                browse_span_start: '<a class="browse-teams" href="">',
-                                search_span_start: '<a class="search-teams" href="">',
-                                create_span_start: '<a class="create-team" href="">',
-                                span_end: '</a>'
-                            }
-                            );
-                        HtmlUtils.append(
-                            self.$el,
-                            HtmlUtils.template(teamActionsTemplate)({message: message})
-                        );
-                    }
-                });
+                this.collection.refresh().done(_.bind(function() {
```

Comment:
what is _.bind()?

Reply:
just binds `this` context

Reply:
For my education, is `_.bind()` any different than the native `bind()`?

Reply:
honestly, this is what I used back when doing es5, and I wasn't sure native bind would function as expected in this env